logging/cloud-client Nightly failures

```
------------------------------------------------------------
- testing logging/cloud-client
------------------------------------------------------------
[ERROR] OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 63.106 s <<< FAILURE! - in com.example.logging.LoggingIT
[ERROR] com.example.logging.LoggingIT.testWriteAndListLogs  Time elapsed: 60.571 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 60000 milliseconds
	at com.example.logging.LoggingIT.testWriteAndListLogs(LoggingIT.java:89)

[ERROR] Errors: 
[ERROR]   LoggingIT.testWriteAndListLogs:89 » TestTimedOut test timed out after 60000 mi...
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0
```

--- a/logging/cloud-client/pom.xml
+++ b/logging/cloud-client/pom.xml
@@ -67,7 +67,7 @@
         <configuration>
           <classesDirectory>${project.build.outputDirectory}</classesDirectory>
           <forkCount>3</forkCount>
-          <argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
+          <argLine>-Xmx1024m</argLine>
         </configuration>
         <executions>
           <execution>

--- a/logging/cloud-client/src/main/java/com/example/logging/ListLogs.java
+++ b/logging/cloud-client/src/main/java/com/example/logging/ListLogs.java
@@ -22,9 +22,7 @@ import com.google.cloud.logging.Logging;
 import com.google.cloud.logging.Logging.EntryListOption;
 import com.google.cloud.logging.LoggingOptions;
 
-/**
- * List logs programmatically using the StackDriver Logging API.
- */
+/** List logs programmatically using the StackDriver Logging API. */
 public class ListLogs {
 
   /** Expects an existing Stackdriver log name as an argument. */
@@ -40,15 +38,13 @@ public class ListLogs {
       String logFilter = "logName=projects/" + options.getProjectId() + "/logs/" + logName;
 
       // List all log entries
-      Page<LogEntry> entries = logging.listLogEntries(
-          EntryListOption.filter(logFilter));
+      Page<LogEntry> entries = logging.listLogEntries(EntryListOption.filter(logFilter));
       do {
         for (LogEntry logEntry : entries.iterateAll()) {
           System.out.println(logEntry);
         }
         entries = entries.getNextPage();
       } while (entries != null);
-
     }
     // [END logging_list_log_entries]
   }

--- a/logging/cloud-client/src/test/java/com/example/logging/LoggingIT.java
+++ b/logging/cloud-client/src/test/java/com/example/logging/LoggingIT.java
@@ -26,7 +26,7 @@ import com.google.cloud.logging.Payload.StringPayload;
 import java.io.ByteArrayOutputStream;
 import java.io.PrintStream;
 import java.util.Collections;
-import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -42,9 +42,9 @@ public class LoggingIT {
 
   private ByteArrayOutputStream bout;
   private PrintStream out;
-  private Logging logging = LoggingOptions.getDefaultInstance().getService();
+  private static Logging logging = LoggingOptions.getDefaultInstance().getService();
 
-  private void deleteLog(String logName) {
+  private static void deleteLog(String logName) {
     logging.deleteLog(logName);
   }
 
@@ -55,8 +55,8 @@ public class LoggingIT {
     System.setOut(out);
   }
 
-  @After
-  public void tearDown() {
+  @AfterClass
+  public static void tearDown() {
     // Clean up created logs
     deleteLog(QUICKSTART_LOG);
     deleteLog(TEST_WRITE_LOG);
@@ -71,7 +71,7 @@ public class LoggingIT {
     assertThat(got).contains("Logged: Hello, world!");
   }
 
-  @Test(timeout = 10000)
+  @Test(timeout = 30000)
   public void testWriteAndListLogs() throws Exception {
     // write a log entry
     LogEntry entry =

--- a/logging/cloud-client/src/test/java/com/example/logging/LoggingIT.java
+++ b/logging/cloud-client/src/test/java/com/example/logging/LoggingIT.java
@@ -32,9 +32,7 @@ import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/**
- * Tests for quickstart sample.
- */
+/** Tests for quickstart sample. */
 @RunWith(JUnit4.class)
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
 public class LoggingIT {
@@ -73,13 +71,14 @@ public class LoggingIT {
     assertThat(got).contains("Logged: Hello, world!");
   }
 
-  @Test(timeout = 60000)
+  @Test(timeout = 10000)
   public void testWriteAndListLogs() throws Exception {
     // write a log entry
-    LogEntry entry = LogEntry.newBuilder(StringPayload.of("Hello world again"))
-        .setLogName(TEST_WRITE_LOG)
-        .setResource(MonitoredResource.newBuilder("global").build())
-        .build();
+    LogEntry entry =
+        LogEntry.newBuilder(StringPayload.of("Hello world again"))
+            .setLogName(TEST_WRITE_LOG)
+            .setResource(MonitoredResource.newBuilder("global").build())
+            .build();
     logging.write(Collections.singleton(entry));
     // flush out log immediately
     logging.flush();

